### PR TITLE
REF/CLN: collect imports at top of file, de-duplicate imports

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

**Comment by bashtage** [comment]
Cleaning of very old, unloved code. 

Ideally, the working examples in sm\sm\examples should be moved to notebooks in sm\examples\notebooks.

**Comment by bashtage** [comment]
4 years since the last non-bugfix change.

File: statsmodels/examples/ex_generic_mle_tdist.py [hunk 1]
```diff
@@ -156,15 +163,6 @@ def nloglikeobs(self, params):
...
 
 
-
-
-
-#import for kstest based estimation
-#should be replace
-# FIXME: importing these patches scipy distribution classes in-place.
-#  Don't do this.
-import statsmodels.sandbox.distributions.sppatch  # noqa:F401
```
Comment (josef-pkt) [inline statsmodels/examples/ex_generic_mle_tdist.py]: don't change this import location
Given that this is monkey patching scipy, the location of the import is important and might change other results.

**Comment by jbrockmendel** [comment]
> Take one example and convert it into a notebook

Awaiting comment on #5795.

**Comment by josef-pkt** [comment]
#5795 add it as or to a notebook for the docs and we don't have to worry about changes to the original script. 
The script file then just needs to be marked for deletion if there is no extra content left.

**Comment by jbrockmendel** [comment]
> add it as or to a notebook for the docs

Should "as or" be "as is or"?  Wouldn't we want to avoid making the currently-wrong version into a notebook?

**Comment by josef-pkt** [comment]
I meant add it `as a new notebook` or add it `to an existing notebook` if that location is appropriate.

Based on skimming your changes, they look good, but I didn't check the details, nor what is "wrong" with the current version.
You put **your** version with cleanup and changes and possibly extra explanations in the notebook, but as in that PR, the existing example script files can be taken as basis for noteboooks.

**Pull request merged**